recipes-devtools/ruby_3.1.0.bb

```diff
@@ -0,0 +1,108 @@
+require ruby.inc
+
+DEPENDS:append:libc-musl = " libucontext"
+
+SRC_URI += " \
```

Comment:
potential issue: Patch '0006-Make-gemspecs-reproducible.patch' should contain a Signed-off-by entry